mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -161,14 +206,22 @@ def __init__(self,
                  reduction='mean',
                  class_weight=None,
                  loss_weight=1.0,
-                 loss_name='loss_ce'):
+                 loss_name='loss_ce',
+                 avg_non_ignore=False):
         super(CrossEntropyLoss, self).__init__()
         assert (use_sigmoid is False) or (use_mask is False)
         self.use_sigmoid = use_sigmoid
         self.use_mask = use_mask
         self.reduction = reduction
         self.loss_weight = loss_weight
         self.class_weight = get_class_weight(class_weight)
+        self.avg_non_ignore = avg_non_ignore
+        if not self.avg_non_ignore:
+            warnings.warn(
+                'Default ``avg_non_ignore`` is False, if you would like to '
+                'ignore certain label and average loss over non-ignore label, '
+                'open ``avg_non_ignore=True`` and set ``ignore_index`` '
+                'in decoder or auxiliary heads config file')
```

Comment:
```suggestion
                'Default ``avg_non_ignore`` is False, if you would like to '
                'ignore certain label and average loss over non-ignore label, '
                'set ``avg_non_ignore=True``')
```

Reply:
May `ignore_index` also be reminded to set in warnings thus users would not leave it out?

Reply:
In config file, `ignore_index` belongs to decoder/auxiliary head but `avg_non_ignore` belongs to loss_decode.